### ISSUE TITLE
GitHub Preview

### ISSUE THREAD

**Issue by Tereseus** [piotr010/prj-rev-bwfs-dasmoto#1]
Github has a new preview feature that makes it easy to view a repository's HTML and CSS in action, it is something that is not enabled on your project and one thing you can do to solve that is to remove the extra level you have with the Dasmoto folder, you can just have the HTML and resource folder be the main part of the project. That will enable an easy preview option.

**Comment by piotr010** [comment]
@Tzuridis thanks for that. Do you mean not having css folder like I had before, right?

**Comment by Tereseus** [comment]
The index.html has to be in the base of your repo

**Comment by piotr010** [comment]
Sorry, I am not sure if get you correctly. Did you mean not using general folder?

**Comment by Tereseus** [comment]
Yes the first files you should see when you open the repo are index.htnl and the css resource folder
